script.js

```diff
@@ -0,0 +1,46 @@
+let books = [];
+
+const addBtn = document.getElementById('addBook');
+const listBox = document.getElementById('list');
+
+function displaybooks() {
+  const book = localStorage.getItem('book');
+  if (book === null) {
+    books = [];
+  } else {
+    books = JSON.parse(book);
+  }
+  let bookscontainer = '';
+  books.forEach((item, index) => {
+    bookscontainer += `<h3 id="">${item.title}</h3>
+            <p id="">${item.auther}</p>
+            <button type="button" onclick="Remove(${index})">Remove</button>`;
+  });
+  listBox.innerHTML = bookscontainer;
+}
+
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const title = document.getElementById('bookName').value;
+  const auther = document.getElementById('author').value;
+  const book = localStorage.getItem('book');
+  if (book === null) {
+    books = [];
+  } else {
+    books = JSON.parse(book);
+  }
+  books.push({
+    title,
+    auther,
+  });
+  localStorage.setItem('book', JSON.stringify(books));
+  displaybooks();
```

Comment:
- [ ] While you did a good job adding books to your app :+1:, as a [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md) you should create a function to add new book with title and author parameters :100: